fdbserver/RestoreMaster.actor.cpp

```diff
@@ -690,7 +689,13 @@ ACTOR static Future<Void> collectBackupFiles(Reference<IBackupContainer> bc, std
 		request.targetVersion = desc.maxRestorableVersion.get();
 	}
 
-	Optional<RestorableFileSet> restorable = wait(bc->getRestoreSet(request.targetVersion));
+	if (g_network->isSimulated()) {
```

Comment:
We can put this message into TraceEvent. It is also useful info and not spammy.

Reply:
Will do.